tsconfig.json

```diff
@@ -19,8 +19,5 @@
     "include": [
         "src/**/*.ts",
         "custom_typings/main.d.ts"
-    ],
-    "exclude": [
-        "node_modules"
     ]
```

Comment:
While I would assume the same, without these lines the `atom-typescript` plugin completely locks up my Atom. I am unable to perform edits in the `.ts` files in this project.